Jenkinsfile

```diff
@@ -75,13 +71,13 @@ node {
       }
       stage('Helm install') {
         withCredentials([
-          string(credentialsId: 'sqsQueueEndpoint', variable: 'sqsQueueEndpoint'),
-          string(credentialsId: 'calculationQueueUrlPR', variable: 'calculationQueueUrl'),
-          string(credentialsId: 'calculationQueueAccessKeyIdListen', variable: 'calculationQueueAccessKeyId'),
-          string(credentialsId: 'calculationQueueSecretAccessKeyListen', variable: 'calculationQueueSecretAccessKey'),
-          string(credentialsId: 'paymentQueueUrlPR', variable: 'paymentQueueUrl'),
-          string(credentialsId: 'paymentQueueAccessKeyIdSend', variable: 'paymentQueueAccessKeyId'),
-          string(credentialsId: 'paymentQueueSecretAccessKeySend', variable: 'paymentQueueSecretAccessKey')
+          string(credentialsId: 'sqs-queue-endpoint', variable: 'sqsQueueEndpoint'),
+          string(credentialsId: 'calculation-queue-url-pr', variable: 'calculationQueueUrl'),
+          string(credentialsId: 'calculation-queue-access-key-id-listen', variable: 'calculationQueueAccessKeyId'),
+          string(credentialsId: 'calculation-queue-secret-access-key-listen', variable: 'calculationQueueSecretAccessKey'),
+          string(credentialsId: 'payment-queue-url-pr', variable: 'paymentQueueUrl'),
+          string(credentialsId: 'payment-queue-access-key-id-send', variable: 'paymentQueueAccessKeyId'),
+          string(credentialsId: 'payment-queue-secret-access-key-send', variable: 'paymentQueueSecretAccessKey')
```

Comment:
I've removed these, and the above ones, because they're not valid anymore (unless I've misunderstood)?